blog-posts/thoughs-on-collaborative-work.md

```diff
@@ -0,0 +1,79 @@
+---
+path: "/blog/thoughts-on-collaborative-work/"
+date: "2022-02-28"
+title: "Thoughts on collaborative work"
+author: Jesús Real
+authorSummary: Senior Frontend Engineer @ Satellytes
+featuredImage: images/thoughs-on-collaborative-work/group-table.jpg
+attribution:
+    creator: Marvin Meyer
+    source: https://unsplash.com/photos/SYTO3xs06fU
+seoMetaText: This article aims to offer an overview about the common beliefs around collaborative work, highlight the (many) (surprising) positive sides of it, and depicts some on-field examples that will either suit you or inspire you to find your own ways on approaching collaborative work. 
```

Comment:
Please shorten, it's too long for SEO. Check via https://metatags.io/ where you can see the text ellipsis:
![Screenshot 2022-02-28 at 11 43 39](https://user-images.githubusercontent.com/1701755/155969826-d027459b-2962-4dee-a6ad-f4be23b4e420.png)



The images are not loading in the branch preview it seems, that's not part of this PR though.

Reply:
That's great feedback, thanks for looking into that 🙏 
Updated, see image below
![Screenshot 2022-03-01 at 09 53 17](https://user-images.githubusercontent.com/8238188/156136848-e2656347-08a0-4391-b3ab-77a187bedb64.png)